### PR TITLE
fix(stub): avoid stub cache for teleport for reactive update

### LINKED ISSUE #2033: Bug: Stubbed Teleport does not update reactively
**Describe the bug**

Ref: https://github.com/vuejs/test-utils/issues/798#issuecomment-1513197973

**To Reproduce**

Here: https://github.com/vuejs/test-utils/issues/798#issuecomment-1513197973

**Expected behavior**

Should update count to 1.

**Related information:**

**Additional context**

### PR DESCRIPTION
**Pull request by freakzlike**
Again an issue with the `transformationCache`. I'm not quite sure yet why the cache is causing these issues.

fix #2033

---
Also tested if https://github.com/vuejs/test-utils/issues/1781 is fixed aswell, but it wasn't the case

### CODE REVIEW

File: src/vnodeTransformers/util.ts [hunk 1]
```diff
@@ -41,12 +41,11 @@ export const createVNodeTransformer = ({
     }
 
     const cachedTransformation = transformationCache.get(originalType)
-    if (cachedTransformation) {
-      // https://github.com/vuejs/test-utils/issues/1829 & https://github.com/vuejs/test-utils/issues/1888
-      // Teleport/KeepAlive should return child nodes as a function
-      if (isTeleport(originalType) || isKeepAlive(originalType)) {
-        return [cachedTransformation, props, () => children, ...restVNodeArgs]
-      }
```
Comment (cexbrayat) [inline src/vnodeTransformers/util.ts:49]: interesting that the workaround to get rid of the warnings is no longer needed 👍

**Review by cexbrayat — APPROVED** [review]

File: src/vnodeTransformers/util.ts (hunk 1 above)
Comment (freakzlike) [reply src/vnodeTransformers/util.ts:49]: It's still needed a few lines below

**Review by lmiller1990 — APPROVED** [review]

**Review by lmiller1990 — COMMENTED** [review]
Great @freakzlike  - mind rebasing this one? Thanks 🙏

**Comment by freakzlike** [comment]
> Great @freakzlike - mind rebasing this one? Thanks 🙏

@lmiller1990 updated 👍

**Pull request merged**